src/settings/toml/manifest.rs

```diff
@@ -235,7 +245,13 @@ impl Manifest {
             }
 
             // don't inherit kv namespaces because it is an anti-pattern to use the same namespaces across multiple environments
-            target.kv_namespaces = get_namespaces(environment.kv_namespaces.clone(), preview)?;
+            target.kv_namespaces = get_kv_namespaces(environment.kv_namespaces.clone(), preview)?;
+
+            // don't inherit actor namespaces because it could be potentially dangerous to use the same namespaces across environments.
+            target.actor_namespaces = get_actor_namespaces(
```

Comment:
based on the comments above... shouldn't site have an entry in this block too? It looks as if it will be inherited currently.